various-receipes/typescript/ts-jest.config.js

```diff
@@ -0,0 +1,5 @@
+const config = require('../../jest.config');
+config.testMatch = ['**/various-receipes/typescript/test/*.ts', '!**/playground/**', '!**/stryker-tmp/**'];
+config.preset = 'ts-jest';

```

Comment:
Since this is for educational purposes, should we explicitly show the config here instead of inheriting from a preset (kinda magic for the reader)?

In few lines, we can show what does it mean for Jest === Set the transformer and probably not much more